.circleci/config.yml

```diff
@@ -50,8 +50,25 @@ jobs:
           path: ~/project
       - attach_workspace:
           at: ~/project
+      - run: git clone git@github.com:redbadger/pride-android-secrets.git
+      - run: mkdir -p .gradle 
+      - run: echo -e "keystore=$ANDROID_KEYSTORE_PATH\nkeystore.alias=$ANDROID_KEYSTORE_ALIAS\nkeystore.password=$ANDROID_KEYSTORE_PASSWORD\nkeystore.keypass=$ANDROID_KEYSTORE_PASSWORD" >> .gradle/gradle.properties
       - run: bundle install
       - run: bundle exec fastlane build
```

Comment:
Does this do the Android build and signing?

Reply:
Yes, we clone the keystore and then populate a properties file with the secrets and path of the keystore. Anyone can create their own properties file with their own keystore for local development. Then we use Fastlane(a Ruby gem) to handle the building of a signed binary.